docs/python/training/conf.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+
+# -- Path setup --------------------------------------------------------------
+
+import os
+import sys
+
+# ORTModule source-code folder
+sys.path.insert(0, os.path.abspath(os.path.join('..','..','..','..','orttraining','orttraining','python','training')))
```

Comment:
may we instead use relative path from the build directory to ```onnxruntime/training```?

Reply:
PATH env var must be absolute path